content/en/docs/ops/ambient/usage/ztunnel/_index.md

```diff
@@ -0,0 +1,498 @@
+2023-08-14T09:33:25.028601Z  INFO inbound{id=1ebc3c7384ee68942bbb7c7ed866b3d9 peer_ip=10.240.2.16 peer_id=spiffe://cluster.local/ns/ambient-demo/sa/sleep}: ztunnel::proxy::inbound: got CONNECT request to 10.240.1.11:80
+
+--snip--
+
+2023-08-14T09:33:25.226403Z  INFO outbound{id=9d99723a61c9496532d34acec5c77126}: ztunnel::proxy::outbound: proxy to 10.240.1.11:80 using HBONE via 10.240.1.11:15008 type Direct
+2023-08-14T09:33:25.273268Z  INFO outbound{id=9d99723a61c9496532d34acec5c77126}: ztunnel::proxy::outbound: complete dur=46.9099ms
+2023-08-14T09:33:25.276519Z  INFO outbound{id=cc87b4de5ec2ccced642e22422ca6207}: ztunnel::proxy::outbound: proxying to 10.240.2.10:80 using node local fast path
+2023-08-14T09:33:25.276716Z  INFO outbound{id=cc87b4de5ec2ccced642e22422ca6207}: ztunnel::proxy::outbound: complete dur=231.892µs
+
+--snip--
+{{< /text >}}
+
+Here we note the logs first indicating the http CONNECT request to the new destination pod (10.240.1.11) which indicates the setup of ztunnel to the node hosting the additional destination service pod. This is then followed by logs indicating the client traffic being sent to both 10.240.1.11 and 10.240.2.280 which are the two destination pods providing the service. Also note that the data path is performing client-side load balancing in this case and not depending on Kubernetes service load balancing. The client pod logs directly show successive requests being sent to the different destination pods and not to the Kubernetes ClusterIP service IP address, indicating the use of client side load balancing in this scenario.
+
+Note: The Ztunnel load balancing algorithm is an internally fixed L4 Round Robin algorithm that distributes traffic based on TCP session state and is not user configurable. 
+
+This is a Round robin load balancing algorithm and is separate from and independent of any load balancing algorithm that may be configured within a VirtualService's TrafficPolicy field, since as discussed previously, all aspects of VirtualService api objects are instantiated on the Waypoint proxies and not the ztunnel proxies. 
+
+TODO any other checks/ monitoring to be added ?
+
+### Pod selection logic for Ambient and Sidecar modes
+Istio with sidecar proxies can co-exist with `ambient` based node level proxies within the same compute cluster. Since there are multiple configuration parameters and resource annotations that govern whether a given pod will be setup to use a sidecar proxy or an ambient node proxy, the logic to make this decision is as follows.
+
+1. The `istio-cni` plugin configuration exclude list configured in `cni.values.excludeNamespaces` is used to skipnamespaces in the exclude list.
+2. `ambient` mode is used for a pod if 
+- The namespace has label "istio.io/dataplane-mode" == "ambient" 
+- The annotation "sidecar.istio.io/status" is not present on the pod
+- "ambient.istio.io/redirection" is not "disabled"
+
```

Comment:
Suggest we move this section up so it comes right after the `#Adding workloads to ambient mesh` section, or maybe include this piece in that section.

Reply:
Retaining the order of this section for now but added a pointer to it in the Enabling Ambient section. Since we want the first pass of users to work with simple pure `ambient` only scenarios, it would be better to complete both deploymewnt and monitoring of pure ambient before getting into mixed scenarios of ambient + sidecar.j